rllib/algorithms/algorithm_config.py

```diff
@@ -332,15 +332,9 @@ def __init__(self, algo_class=None):
 
         # `self.explore()`
         self.explore = True
-        self.exploration_config = {
-            # The Exploration class to use. In the simplest case, this is the name
-            # (str) of any class present in the `rllib.utils.exploration` package.
-            # You can also provide the python class directly or the full location
-            # of your class (e.g. "ray.rllib.utils.exploration.epsilon_greedy.
-            # EpsilonGreedy").
-            "type": "StochasticSampling",
-            # Add constructor kwargs here (if any).
-        }
+        # This is not compatible with RLModules, which have a method
```

Comment:
TODO (Artur): Test which algorithms need `exploration_config`'s default old value and which don't on our ci. Then deprecate their default behavior so that for all the derived algorithms the default becomes empty dict.